streaming/src/test/streamingqueue_tests.cc

```diff
@@ -0,0 +1,65 @@
+#define BOOST_BIND_NO_PLACEHOLDERS
```

Comment:
nit, file name should be `streaming_queue_tests.cc`

Reply:
> nit, file name should be `streaming_queue_tests.cc`

fixed.